src/config/dynamic-theme-fixes.config

```diff
@@ -17956,6 +17956,26 @@ pre {
 
 ================================
 
+pathfinderwiki.com
+
+CSS
+body {
+    background-color: rgb(10,10,10);
```

Comment:
As Dark Reader can be used with Dark or Light scheme forcing dark color break light mode.
To have the same for dark but working fine in light we use ${original color to invert}

So change this line to:
`background-color: ${rgb(245,245,245)};`